server/app/controllers/applicant/UpsellController.java

```diff
@@ -93,7 +96,8 @@ public CompletionStage<Result> considerRegister(
       long applicantId,
       long programId,
       long applicationId,
-      String redirectTo) {
+      String redirectTo,
+      String submitTime) {
```

Comment:
**question:** Why do we pass this through as a string if we convert it back to an instant on line 168? Could we change it to pass it through as an instant (which I think is the type before calling `.toString` on it)?

Reply:
It goes through routes

`controllers.applicant.UpsellController.considerRegister(request: Request, applicantId: Long, programId: Long, applicationId: Long, redirectTo: String, submitTime: String)`

I couldn't figure out how to pass Java objects. Is there a better way?

Reply:
Ah yeah that makes sense

Reply:
This is consistent with how we pass other dates on routes.